packages/Brew/yugabyte.rb

```diff
@@ -0,0 +1,82 @@
+class Yugabyte < Formula
```

Comment:
I think this should be contributed directly to https://github.com/Homebrew/homebrew-core

Reply:
I have created homebrew tap for yugabyte, once tap PR is approved, We will plan to move it to Homebrew core repo.

Reply:
@vp15591 thanks! Would you mind linking to the tap PR? Also I don't think we should put this Homebrew code into the main yugabyte-db repo, even temporarily.